shoop/core/models/_contacts.py

```diff
@@ -359,7 +359,7 @@ def get_person_contact(user):
         'last_name': getattr(user, 'last_name', ''),
         'email': getattr(user, 'email', ''),
     }
-    return PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
+    return PersonContact.objects.filter(is_active=True).get_or_create(user=user, defaults=defaults)[0]
```

Comment:
Yes, it looks like it will. Currently, I think this is the behavior if a contact doesn't exist.

For example, if you create a user from admin, they don't have a contact. However, on their first login a contact is created by this function in middleware that creates the contact. Not sure what is the correct behavior since (at least through our frontend) inactive contacts can't login. Maybe we should add a parameter that determines whether or not inactive contacts should be filtered out.